tools/public_api_guard/material/dialog.md

```diff
@@ -290,6 +291,9 @@ export class MatDialogTitle implements OnInit {
 // @public
 export function throwMatDialogContentAlreadyAttachedError(): void;
 
+// @public
+export const _transitionTime = 150;
```

Comment:
Can we avoid exposing the publicly?

Reply:
I would like to, that's kind of what I was trying to ask in https://github.com/angular/components/pull/23228#pullrequestreview-716273254.